src/js/Accordion.js

```diff
@@ -58,7 +61,7 @@ export default class Accordion {
      */
     initAccordion(accordion, mono) {
         console.log("Accordion instance init()");
-
+        accordion.classList.add("is-initialized");
```

Comment:
@ijurko koji je cilj ovog zapravo?

Reply:
@igorbumba Znaci zapravo mi ovo trebalo kad radimo puno sa ajaxom i moramo ponovo initat neke klase.
I onda bi htio da preskoci ako je vec initano.
Jel ti ima to smisla ?

Reply:
ima smisla, zato sam pitao samo da razumijem kontekst

Reply:
naravno, mislio sam vec da imas mozda bolju ideju. Tnx :)